image/fake-keys/src/lib.rs

```diff
@@ -96,102 +96,102 @@ pub const VENDOR_KEY_3_PRIVATE: ImageEccPrivKey = [
     0xbc900c70, 0xb8cb36ac, 0x268b8079, 0xe3aeaaaf,
 ];
 pub const VENDOR_LMS_KEY0_PRIVATE: ImageLmsPrivKey = ImageLmsPrivKey {
-    tree_type: 0x0c000000,
-    otstype: 0x07000000,
+    tree_type: 0xc000000,
```

Comment:
Could these magic values be const values with human readable names?  or a comment next to them